### PR TITLE
Add required attribute for accessibility

### PR DESCRIPTION
**Pull request by atscott**
The aria-label attribute is required for accessibility on the datepicker button.

### CODE REVIEW

**Comment by atscott** [comment]
I signed it!

File: src/components/datepicker/datePicker.js [hunk 1]
```diff
@@ -54,6 +54,7 @@
           // may be confusing.
           '<md-button class="md-datepicker-button md-icon-button" type="button" ' +
               'tabindex="-1" aria-hidden="true" ' +
+              'aria-label="{{::ctrl.dateLocale.msgOpenCalendar}}" ' +
```
Comment (ThomasBurleson) [inline src/components/datepicker/datePicker.js]: Why is this needed, when aria-hidden == true ?

**Comment by atscott** [comment]
Ah, sorry - didn't notice that. The problem I thought I was fixing was actually a bug in mdAria and was fixed in https://github.com/angular/material/commit/b3cb84d3fa9408fb3f7192020eaef0d2c82bd929
